src/System.IO/src/System.IO.builds

```diff
@@ -10,6 +10,9 @@
       <TargetGroup>net463</TargetGroup>
     </Project>
     <Project Include="System.IO.csproj">
+      <TargetGroup>netcoreapp1.1</TargetGroup>
```

Comment:
@jkotas I don't think that is actually true. The default is netstandard1.7 (see https://github.com/dotnet/corefx/blob/master/src/System.IO/src/System.IO.csproj#L10).  There will need to be a netcoreapp1.1 configuration.